src/Store.js

```diff
@@ -519,6 +520,16 @@ class Store {
   _addOperationBatch (data, batchOperation, lastOperation, onProgressCallback) {
     throw new Error('Not implemented!')
   }
+  
+  _procEntry(entry) {
+    var { payload, hash } = entry;
+    var { op } = payload;
+    if(op) {
+      this.events.emit(`log.op.${op}`, this.address.toString(), hash, payload);
+    } else {
+      this.events.emit(`log.op.none`, this.address.toString(), hash, payload);
```

Comment:
Is there a case when `op` would be empty? I believe that's not possible, or at least shouldn't be.

Reply:
Yeah I agree there shouldn't be a case where op is missing.

Reply:
Should it throw an error or a warning then? Strange things happen every day